Makefile.armv7m

```diff
@@ -49,29 +49,27 @@ else ifeq ($(TARGET_SUBFAMILY), stm32l4x6)
 	VADDR_KERNEL_INIT=8000000
 	KERNEL_TARGET_DEFINE=-DCPU_STM32L4X6 -DNOMMU
 else ifeq ($(TARGET_SUBFAMILY), imxrt105x)
-	RAM_SIZE=128
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT105X -DNOMMU 
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT105X -DNOMMU
 else ifeq ($(TARGET_SUBFAMILY), imxrt106x)
-	RAM_SIZE=256
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT106X -DNOMMU 
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT106X -DNOMMU
 else ifeq ($(TARGET_SUBFAMILY), imxrt117x)
-	RAM_SIZE=256
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT117X -DNOMMU 
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT117X -DNOMMU
 else
 	$(error Incorrect TARGET.)
 endif
 
 
-LDFLAGS = -z max-page-size=0x10 --gc-sections -q
+LDFLAGS = -z max-page-size=0x10 --gc-sections
```

Comment:
`LDFAGS` would better be defined with `:=` as it's static and could be resolved at declaration time

Reply:
Done